app/views/static/english_language_guidance.html.erb

```diff
@@ -0,0 +1,91 @@
+<% content_for :page_title, 'English language guidance' %>
+<% content_for :back_link_url, back_link_url %>
+
+<h1 class="govuk-heading-xl">Use an English language test in your qualified teacher status (QTS) application</h1>
+
+<p class="govuk-body-lead">If you plan to provide evidence that you’ve passed an English language test as part of your QTS application, you’ll need to make sure your test meets all the requirements.</p>
+
+<h2 class="govuk-heading-m">What your English language test must show</h2>
+
+<p class="govuk-body">For QTS, the test must be a ‘Secure English Language test’ (SELT). That means it was taken:</p>
+
+
+<ul class="govuk-list govuk-list--bullet">
+    <li>with an approved provider</li>
+    <li>at an approved test location</li>
+    <li>within the last 2 years</li>
+</ul>
+
+<p class="govuk-body">Your test must be at level B2 or above of the <a
+  href="https://www.coe.int/en/web/common-european-framework-reference-languages/level-descriptions"
+  class="govuk-link">Common European Framework of Reference for Languages (CEFR)</a>.
+</p>
+<h3 class="govuk-heading-m">Approved providers and tests</h3>
+
+<p class="govuk-body">The Teaching Regulation Authority (TRA) recognises 5 approved SELT providers for QTS.</p>
+
+<p class="govuk-body">
+  Each provider offers a number of different tests, so it’s important to understand which ones we can accept.
+</p>
+
+<p class="govuk-body">
+  You’ll also need to know your provider’s required scores for the 4 disciplines of listening, speaking, reading and writing.
+</p>
+
+<table class="govuk-table">
+  <caption class="govuk-table__caption govuk-table__caption--m govuk-visually-hidden">Approved providers and tests</caption>
+  <thead class="govuk-table__head">
+    <tr class="govuk-table__row">
+      <th scope="col" class="govuk-table__header">Approved provider name</th>
+      <th scope="col" class="govuk-table__header">Tests we can accept</th>
+      <th scope="col" class="govuk-table__header">Required scores</th>
+    </tr>
+  </thead>
+
+  <tbody class="govuk-table__body">
+    <% EnglishLanguageProvider.all.each do |provider| %>
+      <tr class="govuk-table__row">
+        <td class="govuk-table__cell"><a href="<%= provider.provider_url %>" class="govuk-link"><%= provider.name %></a></td>
+        <td class="govuk-table__cell"><%= provider.accepted_tests %></td>
+        <td class="govuk-table__cell">
+          Listening:&nbsp;<%= provider.b2_level_requirement %><br>
+          Speaking:&nbsp;<%= provider.b2_level_requirement %><br>
+          Reading:&nbsp;<%= provider.b2_level_requirement %><br>
+          Writing:&nbsp;<%= provider.b2_level_requirement %><br>
+        </td>
+      </tr>
+    <% end %>
+  </tbody>
+</table>
+
+<div class="govuk-warning-text">
```

Comment:
Similarly, could we use the warning text component here too.

https://govuk-components.netlify.app/components/warning-text/#input-erb-warning-text-provided-by-a-block-with-custom-icon-fallback-text